### DISCUSSION TITLE
Error while dp train: OSError: liquidData/type.raw not found.

### DISCUSSION THREAD

**Discussion by wjinjin** [deepmodeling/deepmd-kit#138] [Q&A]
while I use dp train input.json command, it shows OSError: liquidData/type.raw not found. But there is a type.raw in liquidData file. What could be wrong?

**Comment by njzjz** [comment]
Is `liquidData` in your current directory where you run `dp train`? Can you run `ls liquidData/type.raw` in that directory?

**Comment by wjinjin** [comment]
yes, I can. Now I reduce the data number in liquiddata, the error changes into "IndexError: list index out of range". Does this mean there are to much data in liquiddat?

**Comment by njzjz** [comment]
Something went wrong. Can you provide more information about your `input.json` and your data?
BTW, does the example work well in your machine?

**Comment by wjinjin** [comment]
Now I reduce the data number in liquiddata, it's working. 
The input.json is shown as below.
"_comment": " model parameters",
    "model": {
        "descriptor" :{
            "type":             "se_a",
            "sel":              [30],
            "rcut_smth":        2.50,
            "rcut":             3.00,
            "neuron":           [50, 100],
            "resnet_dt":        false,
            "axis_neuron":      16,
            "seed":             1,
            "_comment":         " that's all"
        },
        "fitting_net" : {
            "neuron":           [240, 240, 240],
            "resnet_dt":        true,
            "seed":             1,
            "_comment":         " that's all"
        },
        "_comment":     " that's all"
    },

    "learning_rate" :{
        "type":         "exp",
        "start_lr":     0.001,
        "decay_steps":  15000,
        "decay_rate":   0.95,
        "_comment":     "that's all"
    },

    "loss" :{
        "start_pref_e": 0.02,
        "limit_pref_e": 1,
        "start_pref_f": 1000,
        "limit_pref_f": 1,
"start_pref_v": 0,
        "limit_pref_v": 0,
        "_comment":     " that's all"
    },

    "_comment": " traing controls",
    "training" : {
        "systems":      [
        "liqiudData"
        ],
        "set_prefix":   "set",
        "stop_batch":   3000000,
        "batch_size":   1,
        "seed":         1,
        "_comment": " display and restart",
        "_comment": " frequencies counted in batch",
        "disp_file":    "lcurve.out",
        "disp_freq":    100,
        "numb_test":    10,
        "save_freq":    1000,
        "save_ckpt":    "model.ckpt",
        "load_ckpt":    "model.ckpt",
        "disp_training":true,
        "time_training":true,
 "profiling":    false,
        "profiling_file":"timeline.json",
        "_comment":     "that's all"
    },

    "_comment":         "that's all"
}

**Comment by jameswind** [comment]
What was the original number of data and what's the current number you use?
I still feel it's a bit strange that this caused the problem.
